packages/patternfly-4/react-core/src/components/Alert/examples/WarningAlert.js

```diff
@@ -5,13 +5,14 @@ class WarningAlert extends React.Component {
   render() {
     return (
       <React.Fragment>
-        <Alert variant="warning" title="Warning notification title">
+        <Alert variant="warning" title="Warning notification title" onClose={() => {}}>
```

Comment:
I wonder if we should change this example to a class and actually add the logic to close the alert.  Just a suggestion.   Don't know if we really need it though.

Reply:
I was considering that, but I didn't want to over-complicate the example... I think you're right though, it would be nice to have this. Do you think it should just be something like `state = { showAlertOne: true, showAlertTwo: true }` with `this.closeAlertOne` and `this.closeAlertTwo` methods? I was thinking of doing something that gives names to each alert and having one method to close by name, but that felt even more convoluted for a basic example. I wanted to have a close button on two of the variants to stay consistent with https://pf-next.com/components/Alert/examples/